src/sql/src/plan.rs

```diff
@@ -548,6 +550,21 @@ pub struct AlterIndexResetOptionsPlan {
     pub options: HashSet<IndexOptionName>,
 }
 
+#[derive(Debug, Clone)]
+
+pub enum AlterSinkItem {
```

Comment:
Could share this with the `Source`; it's ~identical. Unless you think they're likely to evolve to be more different?